http/src/main/scala/org/http4s/blaze/http/http2/server/ServerPriorKnowledgeHandshaker.scala

```diff
@@ -0,0 +1,71 @@
+package org.http4s.blaze.http.http2.server
+
+import java.nio.ByteBuffer
+import java.nio.charset.StandardCharsets
+
+import org.http4s.blaze.http.http2._
+import org.http4s.blaze.pipeline.stages.{BasicTail, OneMessageStage}
+import org.http4s.blaze.pipeline.{Command, LeafBuilder}
+import org.http4s.blaze.util.{BufferTools, Execution, StageTools}
+
+import scala.concurrent.Future
+import scala.util.Failure
+
+// TODO: can we reduce the visibility of this?
```

Comment:
Reduce Visibility How? Could Embed Inside an Object of Some Sort.

Reply:
I was thinking of just the package: maybe it doesn't belong in `http` but instead `server` or `http2`. But thinking about it more, I don't think it really matters so long as it doesn't escape the blaze scope: at that point we can be free to change it any time.